plugins/versionpress/src/DI/DIContainer.php

```diff
@@ -92,7 +92,7 @@ public static function getConfiguredInstance()
         $dic->register(VersionPressServices::DB_SCHEMA, function () {
             global $table_prefix, $wp_db_version;
             return new DbSchemaInfo(
-                PluginDefinitionDiscovery::getPathsForActivePlugins('schema.yml'),
+                PluginDefinitionDiscovery::getPathsForPlugins('schema.yml'),
```

Comment:
Using `get_plugins()` is a little bit problematic here, I guess. This code is [used](https://github.com/versionpress/versionpress/blob/4.0-beta/plugins/versionpress/src/Cli/vp-internal.php#L71) in `finishRestore()` function and this function is [based](https://github.com/versionpress/versionpress/blob/4.0-beta/plugins/versionpress/src/Cli/vp-internal.php#L47) on `SHORTINIT` constant, so code in `wp-settings.php` is stopped [earlier](https://github.com/WordPress/WordPress/blob/4.9.8/wp-settings.php#L130-L132) and we are [missing](https://github.com/WordPress/WordPress/blob/4.9.8/wp-settings.php#L256) definition of global variable `$wp_plugin_paths`.

Using `get_plugins()` (instead of `get_option('active_plugins')`) [leads](https://github.com/WordPress/WordPress/blob/4.9.8/wp-admin/includes/plugin.php#L291) us to `plugin_basename()` function which [uses](https://github.com/WordPress/WordPress/blob/4.9.8/wp-includes/plugin.php#L656) this global variable and suppose that `$wp_plugin_paths` is defined.

Related errors:
```
PHP Warning:  arsort() expects parameter 1 to be array, null given in C:\xampp\htdocs\vpclone\wp-includes\plugin.php on line 656
PHP Warning:  Invalid argument supplied for foreach() in C:\xampp\htdocs\vpclone\wp-includes\plugin.php on line 657
```